### ISSUE TITLE
Releases should support software heritage IDs

### ISSUE THREAD

**Issue by neubs-bsi** [eclipse/sw360#942]
### Summary of the Feature

[Software heritage IDs](https://docs.softwareheritage.org/devel/swh-model/persistent-identifiers.html) are valuable data which can also be used to receive the source code of a release. Thus it would be a good idea to support them in SW360. For automation it is necessary that these releases have the valid structure (e.g. `swh:1:rel:<hash>`). This should be enforced by SW360  

### Acceptance Criteria
- [ ] there is a data field for software heritage IDs
- [ ] there is verification that the ID has the correct format
- [ ] there is JS in the UI which supports the user to set the correct ID
- [ ] the software heritage ID is supported via REST
- (optional) the ID is displayed as link in the UI
- (optional) there is functionality to add a source artifact by downloading from software heritage

**Comment by KoukiHama** [comment]
@neubs-bsi 

Hello. I also want to manage software heritage ids in sw360. Fortunately, I found this issue. Is there anything updated? I think my colleagues and I can also contribute to the development of this list.

**Comment by neubs-bsi** [comment]
@KoukiHama 

Hello, thanks for noticing this issue. 
As of now, there is nothing in work in this direction. We would be very grateful to receive a contribution from you and your colleagues regarding this enhancement.

**Comment by KoukiHama** [comment]
Thanks for the reply.I'll look into how to do it.

**Comment by mcjaeger** [comment]
also from our side there are not plans for far this. colleaues and a gsoc student from 2019 has integrated it in FOSSology though.

**Comment by sschuberth** [comment]
Note that currently the source code locations scraped by SWH are quite limited. For example, they do not have "loaders" (as they call it) for source artifacts on Maven Central, or for SVN repos. So if the source coed you look for is not in Git / Mercurial, you might be out of luck finding in on SWH.

**Comment by mcjaeger** [comment]
I agree with the limitation (currently) on SWH to this regard, regaring the SWH reference it could replace the download link maybe. Or, shall we have another (key value) section like URLs in addition to external ids and additional data?
